test/test_123.jl

```diff
@@ -384,18 +384,18 @@ add!(m, Production(X, 0., 1.0, [Output(PFX, :($pwx*$x0))], [Input(PX, x0)] ))
 add!(m, Production(A, 0., :($sigmadm*1.0), [Output(PA, a0, [Tax(ta,GOVT)])], [Input(PD, (d0-cd0)), Input(PM, (m0-cm0))]))
 add!(m, Production(M, 0., 1.0, [Output(PM, m0)], [Input(PFX, :($pwm*$m0/$pm0), [Tax(:($TM*1.), GOVT)])]))
 
-add!(m, DemandFunction(GOVT, 0.,
+add!(m, DemandFunction(HH, :($sigma*1.0),
```

Comment:
With this set up, it matter which order the Consumers are in in the model build.